src/gui/qgstemporalvcrdockwidget.h

```diff
@@ -0,0 +1,176 @@
+/***************************************************************************
+                         qgstemporalvcrdockwidget.h
+                         ---------------
+    begin                : February 2020
+    copyright            : (C) 2020 by Samweli Mwakisambwe
+    email                : samweli at kartoza dot com
+ ***************************************************************************/
+
+/***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+
+#ifndef QGSTEMPORALVCRDOCKWIDGET_H
+#define QGSTEMPORALVCRDOCKWIDGET_H
+
+#include "ui_qgstemporalvcrdockwidgetbase.h"
+
+#include "qgsdockwidget.h"
+#include "qgis_gui.h"
+#include "qgsrange.h"
+
+class QgsMapLayer;
+class QgsTemporalNavigationObject;
+class QgsTemporalMapSettingsWidget;
+class QgsTemporalMapSettingsDialog;
+
+/**
+ * \ingroup gui
+ * The QgsTemporalVcrDockWidget class
+ *
+ * \since QGIS 3.14
+ */
+class GUI_EXPORT QgsTemporalVcrDockWidget : public QgsDockWidget, private Ui::QgsTemporalVcrDockWidgetBase
```

Comment:
Can we drop the "vcr" terminology? I suspect it won't have any meaning for a lot of users/developers. Maybe just go something explicit like "QgsTemporalControllerDockWidget" instead.